lib/direct_select.dart

```diff
@@ -57,14 +69,18 @@ class _DirectSelectState extends State<DirectSelect> {
         top: result + itemSize,
         backgroundColor: widget.backgroundColor,
         child: _MySelectionList(
-          backgroundColor: widget.backgroundColor,
```

Comment:
This was causing additional paint cycles for no reason. The overlay is already painted with backgroundColor.